provisioning/roles/utils/cron/README.md

```diff
@@ -0,0 +1,34 @@
+# Role for cron tasks
+
+This role sets up a task that should execute regularly on the host.
+
+*Note: even though this role is called after the Unix cron daemon, it actually
```

Comment:
No, because we don't intend on deploying Mangaki on non-systemd OSes for the moment, as far as I know.